docs/tutorial/05-IO-Create-Insert-External-Data.ipynb

```diff
@@ -16,11 +16,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    if students_db_file.exists():
</pre><p>Can we use <code>tempfile.TemporaryDirectory</code>? then we shouldn't have do this exists check and unlink (here and above).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/4682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>